src/lib/Command.js

```diff
@@ -47,12 +50,13 @@ class Command {
 		await fs.createFile(path);
 		const textDocument = await workspace.openTextDocument(Uri.file(path));
 		const editor = await window.showTextDocument(textDocument);
-		await editor.insertSnippet(this.generateSnippet(type, options));
+		const fileType = path.endsWith('ts') ? 'ts' : 'js';
```

Comment:
This should use [`path.extname`](https://nodejs.org/dist/latest-v12.x/docs/api/path.html#path_path_extname_path)